torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1152,7 +1153,7 @@ def _get_ignored_params(
             p
             for m in ignored_modules
             for p in m.parameters()
-            if not isinstance(p, FlatParameter)
+            if not _is_flattened(p)
```

Comment:
maybe rename to `_is_fsdp_flattened` or do `import _is_flattened as _is_fsdp_flattened` to provide clarity in case there are other types of flatenning going on in the future.

Reply:
Sounds good. I can rename it to `_is_fsdp_flattened()`.